client/src/components/ReviewFilters/ReviewFilters.test.jsx

```diff
@@ -0,0 +1,99 @@
+    const testFunc = jest.fn();
+    const fakeEvent = { target: { value: 'all' } };
+    const fakeProps = {
+      totalReviewCount: 47,
+      totalPositiveReviewCount: 34,
+    };
+    const component = shallow(<ReviewTypeFilter updateReviewFilters={testFunc} reviewStats={fakeProps} />);
+
+    component
+      .find('.all_reviews_filter')
+      .simulate('change', fakeEvent);
+
+    expect(testFunc).toHaveBeenCalled();
+    expect(component.find('.all_reviews_count_stat').text()).toEqual('(47)');
+    expect(component.find('.positive_reviews_count_stat').text()).toEqual('(34)');
+    expect(component.find('.negative_reviews_count_stat').text()).toEqual('(13)');
+  });
+});
+
+describe('Purchase type filter component', () => {
+  it('Should call the "updateReviewFilters" function upon a change event occurring on one of the radio input elements and should also render the reviewStats passed down to it correctly', () => {
+    const testFunc = jest.fn();
+    const fakeEvent = { target: { value: 'all' } };
+    const fakeProps = {
+      totalReviewCount: 27,
+      purchaseViaSteamCount: 12,
+    };
+    const component = shallow(<PurchaseTypeFilter updateReviewFilters={testFunc} reviewStats={fakeProps} />);
+
+    component
+      .find('.purchase_type_all_filter')
+      .simulate('change', fakeEvent);
+
+    expect(testFunc).toHaveBeenCalled();
+    expect(component.find('.purchase_type_all_count_stat').text()).toEqual('(27)');
+    expect(component.find('.purchase_type_steam_count_stat').text()).toEqual('(12)');
+    expect(component.find('.purchase_type_other_count_stat').text()).toEqual('(15)');
+  });
+});
+
+describe('Language type filter component', () => {
+  it('Should call the "updateReviewFilters" function upon a change event occurring on one of the radio input elements and should also render the reviewStats passed down to it correctly', () => {
+    const testFunc = jest.fn();
+    const fakeEvent = { target: { value: 'all' } };
+    const fakeProps = {
+      totalReviewCount: 12,
+      englishLanguageReviewCount: 4,
+    };
+    const component = shallow(<LanguageTypeFilter updateReviewFilters={testFunc} reviewStats={fakeProps} />);
+
+    component
+      .find('.all_languages_filter')
+      .simulate('change', fakeEvent);
+
+    expect(testFunc).toHaveBeenCalled();
+    expect(component.find('.all_languages_count_stat').text()).toEqual('(12)');
+    expect(component.find('.your_language_count_stat').text()).toEqual('(4)');
+  });
+});
+
+describe('Date range filter component', () => {
+  it('Should call the "updateReviewFilters" function upon a change event occurring on one of the date input elements', () => {
+    const testFunc = jest.fn();
+    const fakeEvent = { target: { value: 'all' } };
+    const component = shallow(<DateRangeFilter updateReviewFilters={testFunc} />);
+
+    component
+      .find('.start_date_input')
+      .simulate('change', fakeEvent);
+
+    expect(testFunc).toHaveBeenCalled();
+  });
+});
```

Comment:
I see you are only using enzyme's shallow rendering which tests each component in isolation. There is also a mount render where you can test parent components and their children to test that they are linked together properly. But I do think the way you are testing each component with its own fakeProps is a good way to go about it and kind of doing testing that relationship, just not directly.